### PR TITLE
Update uninstall script, fixes #205

### LINKED ISSUE #205: Uninstall fails and doesn't attempt to totally clean up
The uninstall script seems to be broken on MBP (10.11.4): 

``` bash
dan@kraken:~
$ ~/.scm_breeze/uninstall.sh
sed: 1: "/Users/dan/.bashrc": extra characters at the end of d command
== Removed SCM Breeze from '/Users/dan/.bashrc'
sed: 1: "/Users/dan/.zshrc": extra characters at the end of d command
== Removed SCM Breeze from '/Users/dan/.zshrc'
```

Moreover, running the uninstall script doesn't attempt to remove the scm_breeze configuration files placed within `~` on installation:

``` bash
dan@kraken:~
$ ls -al | grep scm
-rw-r--r--    1 dan   staff   4064 May 21 14:59 .git.scmbrc
drwxr-xr-x   14 dan   staff    476 May 21 14:58 .scm_breeze
-rw-r--r--    1 dan   staff    607 May 21 14:59 .scmbrc
```

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
-for rc in bashrc zshrc; do
-  sed -i '/scm_breeze/d' "$HOME/.$rc"
-  printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
-done
+
+if [[ $OSTYPE == "Darwin" ]]; then 
+  for rc in bashrc zshrc; do
+    sed -i '' '/scm_breeze/d' "$HOME/.$rc"
+    printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
+  endfor
```
Comment (ghthor) [inline uninstall.sh]: Don't think you meant to leave this in? That isn't valid sh that I know about. =D

File: uninstall.sh [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+  done
+else
+if [[ $OSTYPE == "linux-gnu" ]]; then 
+  for rc in bashrc zshrc; do
+    sed -i '' '/scm_breeze/d' "$HOME/.$rc"
```
Comment (ghthor) [inline uninstall.sh]: This line needs to be same as before, yeah?

File: uninstall.sh [hunk 3]
```diff
@@ -1,7 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 # uninstall by (github: bernardofire)
 # Remove line from bashrc and zshrc if present.
-for rc in bashrc zshrc; do
-  sed -i '/scm_breeze/d' "$HOME/.$rc"
-  printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
-done
+
+if [[ $OSTYPE == "Darwin" ]]; then
```
Comment (ghthor) [inline uninstall.sh]: Needs to be `== "darwin"*`, on my shell `echo $OSTYPE` reports `darwin16`. See http://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script

**Comment by chrisfinazzo** [comment]
Ah, you're right - will update.

File: uninstall.sh (hunk 1 above)
Comment (ndbroadbent) [reply uninstall.sh]: +1, the for loop syntax in bash is `for ... do ... done`

**Pull request closed without merging**